src/components/AddNewTodoForm.tsx

```diff
@@ -28,9 +28,11 @@ const AddNewTodoSchema = Yup.object().shape({
     .required(i18n.t("validation:requied")),
 });
 
-interface Props {}
+interface Props {
+  closeAfterAdd: () => void;
```

Comment:
`onSubmit?: () => void ` maybe it will be more universal. What do u think ?

Reply:
I think you are right.